SalesforceCommerceCloud/pwa-kit
# W-14338113: Fix a11y color contrast

# Description


This PR fixes a11y insufficient contrast issues in [W-14338113](https://gus.lightning.force.com/a07EE00001cq6JTYAY)

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Check out page
- Addresses page

# How to Test-Drive This PR

- Check out code
- Add an item to cart
- Go to check out page
- Run axe tool on checkout 
- Verify there is no more insufficient contrast errors

- Go to addresses page
- Add an addresses to show the form
- Click save to show all the error
- Run axe tool for the page
- Verify there is no more insufficient contrast errors on the page

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
